### PR TITLE
Add more sequence expression tests.

### PR DESCRIPTION
**Pull request by mattcoley**
Right now it is difficult to use `selectattr` or `if val is` for set functions. This adds three new expression tests: `containing`, `containingall`, and `within`.

- `containing`: if the sequence contains an element. `if [1, 2, 3] is containing 2)`.
- `containingall`, if the sequence contains all elements of another sequence. `if [1, 2, 3] is containingall [1, 2]`.
- `within`: if a value is in a sequence of elements. `if 1  is within [1, 2, 3])`.

### CODE REVIEW

**Comment by mattcoley** [comment]
We have no unit tests right now for these expression tests. Maybe I should just bite the bullet and write all of them? @boulter ?

File: src/main/java/com/hubspot/jinjava/lib/exptest/IsContainingAllExpTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        return false;
+      }
+    }
+
+    return true;
+  }
+
+  @Override
+  public String getName() {
+    return "containingall";
```
Comment (boulter) [inline src/main/java/com/hubspot/jinjava/lib/exptest/IsContainingAllExpTest.java]: "containsall"?

**Comment by mattcoley** [comment]
Ok @boulter tests are added for these new expression tests.

File: src/main/java/com/hubspot/jinjava/lib/exptest/IsContainingAllExpTest.java (hunk 1 above)
Comment (mattcoley) [reply src/main/java/com/hubspot/jinjava/lib/exptest/IsContainingAllExpTest.java]: Whereas `is containing`, `is containingall`, and `is within` are the best verbs I have thought of.

Comment (mattcoley) [reply src/main/java/com/hubspot/jinjava/lib/exptest/IsContainingAllExpTest.java]: Or we could reserve the keyword `does` and do `if val1 does contain val2` :)

Comment (mattcoley) [reply src/main/java/com/hubspot/jinjava/lib/exptest/IsContainingAllExpTest.java]: But that seems like overkill.

**Pull request merged**